tool/lib/commands/release_notes_helper.dart

```diff
@@ -0,0 +1,238 @@
+// Copyright 2024 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+
+import 'package:args/command_runner.dart';
+import 'package:cli_util/cli_logging.dart';
+import 'package:devtools_tool/model.dart';
+import 'package:devtools_tool/utils.dart';
+import 'package:io/io.dart';
+import 'package:path/path.dart' as p;
+
+class ReleaseNotesCommand extends Command {
+  ReleaseNotesCommand() {
+    argParser.addOption(
+      _websiteRepoPath,
+      abbr: 'w',
+      help: 'The absolute path to the flutter/website repo clone on disk.',
+    );
+  }
+
+  static const _websiteRepoPath = 'website-repo';
+
+  @override
+  String get description =>
+      'Creates a PR on the flutter/website repo with the current release notes.';
+
+  @override
+  String get name => 'release-notes';
+
+  @override
+  FutureOr? run() async {
+    final log = Logger.standard();
+    final processManager = ProcessManager();
+
+    final devToolsReleaseNotesDirectory = Directory(
+      p.join(
+        DevToolsRepo.getInstance().devtoolsAppDirectoryPath,
+        'release_notes',
+      ),
+    );
+    final devToolsReleaseNotes = _DevToolsReleaseNotes.fromFile(
+      File(p.join(devToolsReleaseNotesDirectory.path, 'NEXT_RELEASE_NOTES.md')),
+    );
+    final releaseNotesVersion = devToolsReleaseNotes.version;
+    log.stdout(
+      'Drafting release notes for DevTools version $releaseNotesVersion...',
+    );
+
+    // Create a local branch on the flutter/website repo.
+    final websiteRepoPath = argResults![_websiteRepoPath] as String;
+    try {
+      await processManager.runAll(
+        commands: [
+          CliCommand.git(['checkout', '.']),
```

Comment:
```suggestion
          CliCommand.git(['stash']),
```
It might be more polite to stash any changes.